app-modules/ai/src/Actions/SendMessage.php

```diff
@@ -45,7 +45,7 @@
 
 class SendMessage
 {
-    public function __invoke(AiThread $thread, string $content): Closure
+    public function __invoke(AiThread $thread, array $data): Closure
```

Comment:
What do you think about passing the content and files in two separate parameters, each with their own type?